erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-grievance.md

```diff
@@ -0,0 +1,41 @@
+<!-- add-breadcrumbs -->
+# Employee Grievance
+
+> Introduced in version-13
+
+**An employee grievance is a concern, problem, or complaint raised by an employee with what he expects from the company(or their individuals) and its management. Something has made them feel dissatisfied, and they believe it is unfair and/or unjust to them. It may or may not be justified but needs to be tackled very carefully.**
+
+In ERPNext, you can handle these Employee Grievances to make your employees happy.
+
+To access Employee Grievance, go to:
+
+> Home > Human Resources > Employee Grievance
+
+## 1. Prerequisites
+
+Before creating an Attendance record, it is advised that you create the following first:
+
+* [Employee](/docs/user/manual/en/human-resources/employee)
+* [Grievance Type](/docs/user/manual/en/human-resources/grievance-type)
+
+## 2. How to create Employee Grievance
+
+1. Go to Employee Grievance List, click on Add Employee Grievance.
+1. Select Raised by, date, etc.
+1. Fill Grievance Details like Grievance Against Party, Grievance Against, Grievance Type, etc.
+1. Enter Description.
+1. Save your document.
+
+<img class="screenshot" alt="Employee Grievance" src="{{docs_base_url}}/v13/assets/img/human-resources/employee-grievance.png">
+
+> Note: Employee can also link Documents related to the grievance.
```

Comment:
Employees can also link Documents related to the grievance.

Reply:
done